.github/workflows/workflow-markdown-check.md

```diff
@@ -1,4 +1,7 @@
 ## Reusable Workflow for Standardization Validation
 
    - **Purpose:** Validate the organization's standards across projects.
-   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
\ No newline at end of file
+   - **Steps**
+     1. Check for markdown dead links
+     2. Validate line 80 wrap
+   - **Usage:** Call this workflow while making sure the necessary secrets are inherited for successful execution.
```

Comment:
what secrets are necessary to check markdown?

Reply:
removed